dist/cms.js

```diff
@@ -9,9 +9,36 @@ function hasWordPressEmbedBlock() {
   return !!document.querySelector('figure.wp-block-embed');
 }
 
+// Count the number of WordPress embed blocks on the page, including a breakdown by type
+function getWordPressEmbedBlockCounts() {
+  const embedBlocks = document.querySelectorAll('figure.wp-block-embed');
+  const embedsByType = [];
+  for (let embed of embedBlocks) {
+    let embedClasses = embed.className.split( ' ' );
+
+    // Find the provider classname, which starts with "is-provider-".
+    let provider = embedClasses.find( function( className ) {
```

Comment:
This will return the first match. Is it ever possible for the same block to have multiple `is-provider` values?

Reply:
the figure can only contain a single `is-provider-*` class, it is added on save here: https://github.com/WordPress/gutenberg/blob/74d1b0db4732f23085e788eb895f308b92c16aff/packages/block-library/src/embed/save.js#L24